src/test/kotlin/lotto/service/LottoApplicationTest.kt

```diff
@@ -0,0 +1,63 @@
+package lotto.service
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.collections.shouldContain
+import lotto.util.InputModule
+import lotto.util.OutPutModule
+import lotto.util.RandomGenerate
+
+class LottoApplicationTest : DescribeSpec({
+
+    it("로또를 구매하고 당첨 금액을 통계로 알 수 있다") {
+        // given
+        val buyLottoNumbers = listOf<List<Int>>(
+            listOf(1, 2, 3, 4, 5, 6),
+            listOf(1, 2, 3, 4, 5, 7),
+            listOf(1, 2, 3, 4, 5, 8),
+            listOf(1, 2, 3, 4, 15, 8),
+            listOf(1, 2, 3, 14, 15, 8),
+            listOf(1, 2, 13, 14, 15, 8),
+            listOf(1, 12, 13, 14, 15, 8),
+            listOf(11, 12, 13, 14, 15, 8),
+        )
+
+        val stubInputModule: InputModule = object : InputModule {
+            val MONEY_STRING = (buyLottoNumbers.size * 1000).toString()
+            val WINNING_NUMBER_STRING = "1, 2, 3, 4, 5, 6"
+            val BONUS_NUMBER_STRING = "7"
+            val inputStore = mutableListOf(MONEY_STRING, WINNING_NUMBER_STRING, BONUS_NUMBER_STRING)
+            override fun read(): String {
+                return inputStore.removeFirst()
+            }
+        }
+        val stubOutPutModule = object : OutPutModule {
+            val writeStore = mutableListOf<String>()
+            override fun write(outputValue: String) {
+                writeStore.add(outputValue)
+            }
+        }
+        val stubRandomGenerate = object : RandomGenerate {
+            var index = 0
+            override fun makeRandomUniqueIntList(listSize: Int, randomRange: IntRange): List<Int> {
+                return buyLottoNumbers[index++]
+            }
+        }
+        val lottoApplication = LottoApplication(stubInputModule, stubOutPutModule, stubRandomGenerate)
+
+        // when
+        lottoApplication.run()
+
+        // then
+        stubOutPutModule.writeStore shouldContain "8를 구매했습니다\n" +
+            "[1, 2, 3, 4, 5, 6]\n" +
+            "[1, 2, 3, 4, 5, 7]\n" +
+            "[1, 2, 3, 4, 5, 8]\n" +
+            "[1, 2, 3, 4, 8, 15]\n" +
+            "[1, 2, 3, 8, 14, 15]\n" +
+            "[1, 2, 8, 13, 14, 15]\n" +
+            "[1, 8, 12, 13, 14, 15]\n" +
+            "[8, 11, 12, 13, 14, 15]\n"
+        stubOutPutModule.writeStore shouldContain "6개 일치 (2000000000원)- 1개\n5개 일치, 보너스 볼 일치 (30000000원)- 1개\n5개 일치 (1500000원)- 1개\n4개 일치 (50000원)- 1개\n5개 일치 (1500000원)- 1개\n3개 일치 (5000원)- 1개"
```

Comment:
요거 개행때문에 작성하실때 꽤 고통받으셨을것 같은데..!!
코틀린에서는 이렇게 표현하시면 편리합니다~!
```suggestion
        stubOutPutModule.writeStore shouldContain """
                8를 구매했습니다
                [1, 2, 3, 4, 5, 6]
                [1, 2, 3, 4, 5, 7]
                [1, 2, 3, 4, 5, 8]
                [1, 2, 3, 4, 8, 15]
                [1, 2, 3, 8, 14, 15]
                [1, 2, 8, 13, 14, 15]
                [1, 8, 12, 13, 14, 15]
                [8, 11, 12, 13, 14, 15]
                
            """.trimIndent()
        stubOutPutModule.writeStore shouldContain """
                6개 일치 (2000000000원)- 1개
                5개 일치, 보너스 볼 일치 (30000000원)- 1개
                5개 일치 (1500000원)- 1개
                4개 일치 (50000원)- 1개
                5개 일치 (1500000원)- 1개
                3개 일치 (5000원)- 1개
            """
            .trimIndent()
```